verilog/analysis/verilog_linter.cc

```diff
@@ -233,6 +246,9 @@ LinterConfiguration LinterConfigurationFromFlags() {
   const auto& rules = absl::GetFlag(FLAGS_rules);
   config.UseRuleBundle(rules);
 
+  // Apply external waivers
+  config.external_waivers = absl::GetFlag(FLAGS_waivers_file);
```

Comment:
Let's make a TODO note to move all of these calls to `GetFlag` outside of these functions, which will help with test-ability.

Reply:
Sure, added a TODO above.